openmetadata-ui/src/main/resources/ui/src/constants/DataInsight.constants.ts

```diff
@@ -73,42 +74,42 @@ export const DATA_INSIGHT_TAB = {
 export const DAY_FILTER = [
   {
     value: 7,
-    label: 'Last 7 Days',
+    label: i18n.t('label.day-label', { day: 7 }),
   },
   {
     value: 14,
-    label: 'Last 14 Days',
+    label: i18n.t('label.day-label', { day: 14 }),
```

Comment:
label.day-label --> label.last-no-of-days

Reply:
Done.